build.gradle.kts

```diff
@@ -41,6 +42,27 @@ subprojects {
     }
 }
 
+/**
+ * Kotlin binary compatibility validator configuration
+ * https://github.com/Kotlin/binary-compatibility-validator#setup
+ */
+apiValidation {
+    /* Packages that are excluded from public API dumps even if they contain public API. */
+    // ignoredPackages.add("kotlinx.coroutines.internal")
+
+    /* Sub-projects that are excluded from API validation */
+    ignoredProjects.addAll(listOf("sample", "android", "common", "test-utils"))
```

Comment:
We could include test-utils, it's a collection of utils that can be / is used from outside trikot.